catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswRecordMetacardType.java

```diff
@@ -396,16 +396,14 @@ private void addCswBriefRecordAttributes() {
         descriptors.add(new AttributeDescriptorImpl(CSW_TITLE, NON_QUERYABLE /* indexed */, true /* stored */,
                 false /* tokenized */, true /* multivalued */, BasicTypes.STRING_TYPE));
 
-        descriptors
-                .add(new AttributeDescriptorImpl(CSW_ALTERNATIVE, NON_QUERYABLE /* indexed */, true /* stored */,
-                        false /* tokenized */, true /* multivalued */, BasicTypes.STRING_TYPE));
```

Comment:
I think the idea was just to mark the constructor arguments (not their actual value) for convenience.  BasicTypes.java also follows this pattern.
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java.
I started marking them with not, and it makes the descriptors harder to read.